app/views/NewMessageView.js

```diff
@@ -133,11 +133,13 @@ class NewMessageView extends React.Component {
 	}
 
 	createChannel = () => {
+		logEvent(events.CREATE_CHANNEL_START);
```

Comment:
It should identify the screen where the action started: `NEW_MESSAGE_CREATE_CHANNEL`

For example, we have tons of ways to create discussions and we want to know from where the action was triggered.
I saw on the upcoming PRs you're reusing actions, which is wrong.